src/tlo/cli.py

```diff
@@ -826,5 +828,68 @@ def add_tasks(batch_service_client, user_identity, job_id,
     batch_service_client.task.add_collection(job_id, tasks)
 
 
+@cli.command()
+@click.argument(
+    "output_results_directory",
+    type=click.Path(exists=True, file_okay=False, writable=True, path_type=Path),
+)
+@click.argument(
+    "additional_result_directories",
+    nargs=-1,
+    type=click.Path(exists=True, file_okay=False, path_type=Path),
+)
+def combine_runs(
+    output_results_directory: Path, additional_result_directories: tuple[Path]
+) -> None:
+    """Combine runs from multiple batch jobs locally.
+    
+    Merges runs from each draw in one or more additional results directories in
+    to corresponding draws in output results directory.
+    
+    All results directories must contain same draw numbers and the draw numbers
+    must be consecutive integers starting from 0. All run numbers in the output 
+    result directory draw directories must be consecutive integers starting
+    from 0.
+    """
+    if len(additional_result_directories) == 0:
+        msg = "One or more additional results directories to merge must be specified"
+        raise click.UsageError(msg)
+    results_directories = (output_results_directory,) + additional_result_directories
+    draws_per_directory = [
+        sorted(
+            int(draw_directory.name) for draw_directory in results_directory.iterdir()
+        )
+        for results_directory in results_directories
+    ]
```

Comment:
Hi Matt, thanks so much again for this super helpful function of combining the results folders. Would like to let you know that I have run the script <script src="https://gist.github.com/matt-graham/b684a86e21084a5a1c74844e6b98268b.js"></script> you have kindly shared with me and found that the same lines as here would fail because the automatically downloaded .json file is also in the directory. After I deleted the json files in the directories, the code combined the files successfully.

Reply:
Thanks @BinglingICL - I'd forgot about the JSON files as I was only testing on outputs of local runs, but definitely it will be worth updating the script to allow for there being JSON files present!

Reply:
Matt, will you make this change to this PR?

Reply:
Yep will work on this now

Reply:
I've now updated to skip any subpaths which are not directories when iterating over run / draw directories which should hopefully fix issue cause by draws JSON files.